src/content/app/genome-browser/hooks/useGenomeBrowser.ts

```diff
@@ -405,6 +406,26 @@ const useGenomeBrowser = () => {
         }
       });
     }
+
+    if (trackIdToSend === 'focus' && !isTurnedOn) {
+      genomeBrowser?.send({
+        type: OutgoingActionType.SET_VISIBLE_TRANSCRIPTS,
+        payload: {
+          track_id: 'focus',
+          transcript_ids: []
+        }
+      });
+    }
```

Comment:
I don't like this. `trackIdToSend === 'focus'` does not have to refer to focus gene; there will be other focus objects in the focus track as well. Plus, we could just fire the `updateTranscriptTracks` action from the track panel gene component.

Reply:
✅